src/design/palette.css

```
@@ -0,0 +1,19 @@
+:root {
+  --blue-light: #9f9dc2;
+  --blue-dark: #4d478b;
+  --grey: #d2d5dc;
+  --grey-light: #dedede;
```

Existe un bajo contraste entre los iconos y el fondo. ¿Es posible considerar un cambio de color?

```suggestion
  --grey-light: #757575;
```

---

acomodamos un poco mejor las paletas en un file de variables css  ***c6231c109871a2ca93e66bfa807f4bb0e7f23644***